src/omotes_rest/apis/job.py

```diff
@@ -26,18 +31,27 @@
 )
 
 
+class OmotesRestApp(Flask):
```

Comment:
Could this class be moved to `main.py`?

Reply:
Not to `main,py` due to circular imports, but moved it to its own file and removed the duplicate cast! https://github.com/Project-OMOTES/omotes-rest/pull/15/commits/6fd83b8663dad575d9dd282151e385bf5e13e87b